tin/templates/base.html

```diff
@@ -8,6 +8,10 @@
   {% include "meta.html" %}
 
   <link rel="stylesheet" href="{% static 'css/base.css' %}">
+  {% if request.user.dark_mode %}
+    <link rel="stylesheet" href="{% static 'css/dark/base.css' %}">
+    <link rel="stylesheet" href="{% static 'css/dark/edit.css' %}">
```

Comment:
`dark/edit.css` should only be loaded wherever `edit.css` is loaded. The way I suggest doing this is adding a context processor that adds the context variable `dark_mode_enabled` to every template, and use that to optionally load themes.
More information here
https://docs.djangoproject.com/en/4.2/ref/templates/api/#using-requestcontext.
Then, wherever `edit.css` is loaded, just do
```django
{% if dark_mode_enabled %}
  {# link dark/edit.css #}
{% endif %}
```